final/final_setup.R

```diff
@@ -115,26 +234,87 @@ table(bankloans_existing$default, model$fitted.values  > 0.5)
 #    Create a function that would calculate Specifity and Sensitivity for a the same grid as in part 3. The function should output
 #    the threshold value for which the absolute difference is minimized
 
-
+i=0.05 #probability
```

Comment:
these few lines could be written inside the function (including the next one) right before for loop